internal/alloydb/refresh.go

```diff
@@ -97,7 +104,14 @@ func fetchEphemeralCert(
 	if err != nil {
 		return certChain{}, err
 	}
-	resp, err := cl.GenerateClientCert(ctx, inst.project, inst.region, inst.cluster, buf.Bytes())
+	req := &alloydbpb.GenerateClientCertificateRequest{
+		Parent: fmt.Sprintf(
+			"projects/%s/locations/%s/clusters/%s", inst.project, inst.region, inst.cluster,
```

Comment:
`inst.String()` should produce the same result.

Reply:
as discussed offline, `inst.String()` does not return the same result, it excludes the words `projects` , `locations` etc.